php_gearman.c

```diff
@@ -3613,10 +3613,6 @@ static void gearman_client_obj_free(zend_object *object) {
 	zval_dtor(&intern->task_list);
 
 	zend_object_std_dtor(&intern->std);
-
-	// I'm including it because it cleans up a memory leak, but still not sure why it's not automatically
-	// cleaned up like other objects. Need to dig in more
-	efree(intern);
```

Comment:
I was checking this and I believe PHP should be handling it: https://wiki.php.net/phpng-upgrading#zend_object_handlers 

I will do some tests with Valgrind later and see if we are leaking any more.